manager/src/main/resources/define/app-http_sd.yml

```diff
@@ -92,7 +92,7 @@ metrics:
     protocol: httpsd
     httpsd:
       host: ^_^host^_^
-      port: ^_^port^_^
+      port: 1234
```

Comment:
hi, why pined the port number 1234

Reply:
this is the random number. only example.

Reply:
can not use a example number instead of ^_^port^_^ param